ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -1037,6 +1064,23 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: jupyter-datascience-anaconda-python-3.8-test-e2e
+      cli: latest
+      commands: |
+        oc new-project jupyter-datascience-anaconda-python-3-8
+        IFS=':' read -r -a NOTEBOOK_IMAGE <<< "${JUPYTER_MINIMAL_IMAGE}"
```

Comment:
Change the ${JUPYTER_MINIMAL_IMAGE} -> ${JUPYTER_ANACODA_IMAGE}. Also to line 1077